scripts/test_caddy_console_auth_origin.py

```
@@ -0,0 +1,390 @@
+INTERNAL_ONLY_PREFIXES = ["/rest/v1", "/storage/v1"]
+
+
+def strip_comments(text: str) -> str:
+    """Drop whole-line comments and fold backslash continuations.
+
+    Both matter. A commented-out directive must not satisfy a check, and this
+    file's own comment block names every upstream and every port these checks
+    look for, so reading it unstripped makes most of them vacuously true.
+    """
+    folded = re.sub(r"\\\n\s*", " ", text)
+    return "\n".join(l for l in folded.splitlines() if not l.strip().startswith("#"))
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)